### DISCUSSION TITLE
Change Charset

### DISCUSSION THREAD

**Discussion by DerT94** [facebook/docusaurus#3697] [Q&A]
Hey,

Is there a way to change the character set in docusaurus to UTF-8? So that it can display German umlauts?

**Editor**
![image](https://user-images.githubusercontent.com/58786605/98254329-71c3df80-1f7c-11eb-8ca8-954f1ccca020.png)

**Website**
![image](https://user-images.githubusercontent.com/58786605/98254360-7b4d4780-1f7c-11eb-8434-c39f2b7c917c.png)

**Comment by slorber** [comment]
Didn't try but you can look at these:

https://v2.docusaurus.io/docs/theme-classic/#metadatas
https://v2.docusaurus.io/docs/docusaurus-core#head

**Comment by DerT94** [comment]
Thank you

https://v2.docusaurus.io/docs/theme-classic/#metadatas

![image](https://user-images.githubusercontent.com/58786605/98339005-57851280-200b-11eb-90b4-725e3269eeac.png)
Leads to an error at start
![image](https://user-images.githubusercontent.com/58786605/98339102-7b485880-200b-11eb-9207-706c0d31067e.png)

---

https://v2.docusaurus.io/docs/docusaurus-core#head

It works with the following code in the Markdown document

```
//...

import Head from '@docusaurus/Head';

<Head>
    <meta charSet="utf-8" />
</Head>

//...
```

This solution is enough for me

**Reply by slorber** [reply]
great

you'll have to add it to all md docs though.

I'll probably add a charSet feature with the i18n support because all i18n site version may need a different charset

**Reply by slorber** [reply]
A workaround to apply the charset. to all your site might be to swizzle the layout component and use Head there

See for example:

https://v2.docusaurus.io/docs/using-themes/#for-site-owners
